### DISCUSSION TITLE
iMac 10.1 stuck on boot

### DISCUSSION THREAD

**Discussion by marcostuppini** [elementary/triage#200] [Q&A]
Getlemen,
I installed Elementary OS on my old iMac 27" model 10.1, 16-500Gb.
Used for a while but from time to time, it freeze with a dark screen.
After few restart, the Boot does not succeed anymore, only a minus sign "-" appears.
A standard windows keyboard is attached, I cannot enter in the Apple Boot selector (normally is used the "ALT" key during the startup)...
No keys seems to work.
Please help!

Marco smarc@live.it

**Comment by verite72** [comment]
try with CTRL F7 but if it's HDD bad cluster you have to make a Hard drive repair task. if Grub boot ... all is possible. give information of verbose boot. by by
